app/assets/stylesheets/application/common.scss

```diff
@@ -73,8 +72,16 @@ header .container-fluid {
   nav.navbar.navbar-default {
     border: none;
 
-    li.no-image a {padding-top: 20px;}
-
+    li.no-image a {
+      padding-top: 20px;
+      font-size: 120%;
+      border-bottom: 4px solid #3E4659;
+      padding-bottom: 15px;
+      transition: border-bottom .2s;
+    }
+    li.no-image a:hover {
```

Comment:
you can nest it in previous statement using `&:hover`

Reply:
Ok I will do that.